x/chainlink/keeper/feed.go

```diff
@@ -0,0 +1,30 @@
+package keeper
+
+import (
+	"github.com/ChainSafe/chainlink-cosmos/x/chainlink/types"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (k Keeper) CreateFeed(ctx sdk.Context, feed types.MsgFeed) {
+	// TODO: add more complex feed validation here such as verify against other modules
+
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.FeedKey))
```

Comment:
why do we used `prefix.NewStore` here instead of just `ctx.KVStore(k.storeKey)` here?